charts/tenant-base/chart/templates/bucketclaim.yaml

```
@@ -0,0 +1,10 @@
+{{- range .Values.bucketClaims }}
+---
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: {{ printf "%s-%s" .name "bucket" }}
```

This could cause a name conflict, any reason for not including `chart.fullname`? Ex:
```suggestion
  name: {{ printf "%s-%s" (include "chart.fullname" $) .name }}
```

---

The reason I did not do this was for the sake of understandability towards the user, it would be easier for the user to understand what `<name>-bucker` is then `<release-name>-<chart-name>-<name>`, this is important because they would have to mount these them self, on the deployments they want to use the bucket in.
It also would only fix conflictes with different releases within the same namespace, not within the same release.

---

Shouldn't it just be abstracted? The user shouldn't worry about the name, we could just set the relevant AWS_S3 environment variables.

---

So we should just mount all configMaps and secrets to all deployments?